### PR TITLE
[DOC] Add documentation for reconstruction method option

### LINKED ISSUE #1168: Clarify in the documentation of `PETLinear` how to specify the reconstruction method
Related to #1167

### PR DESCRIPTION
**Pull request by NicolasGensollen**
Closes #1168

### CODE REVIEW

File: docs/Pipelines/PET_Introduction.md [hunk 1]
```diff
@@ -35,6 +35,25 @@ bids
...
+There exists four reserved values:
+
+- `acdyn`: for reconstructions with attenuation correction of dynamic data
+- `acstat`: for reconstructions with attenuation correction of static data
+- `nacdyn`: for reconstructions without attenuation correction of dynamic data
+- `nacstat`: for reconstructions without attenuation correction of static data
+
+In addition to these values, when working with ADNI data, the following values are possible:
+
+- `coregdyn`: Corresponds to "ADNI processing steps 1"
```
Comment (AliceJoubert) [inline docs/Pipelines/PET_Introduction.md:52]: What do you call ADNI processing steps ?

File: docs/Pipelines/PET_Linear.md [hunk 2]
```diff
@@ -37,36 +31,35 @@ clinica run pet-linear [OPTIONS] BIDS_DIRECTORY CAPS_DIRECTORY ACQ_LABEL
...
+- The reference region is used to perform intensity normalization (i.e. dividing each voxel of the image by the average uptake in this region) resulting in a standardized uptake value ratio ([SUVR](../glossary.md#suvr)) map.
+  It can be `cerebellumPons` or `cerebellumPons2` (used for amyloid tracers) and `pons` or `pons2` (used for FDG).
+  See [PET introduction](./PET_Introduction.md) for more details about masks versions.
+
+By default, cropped images (matrix size 169×208×179, 1 mm isotropic voxels) are generated to reduce the computing power required when training deep learning models.
+Use the `--uncropped_image` option if you do not want to crop the image.
+
+It is possible to select only images based on a [specific reconstruction method](./PET_Introduction.md#reconstruction-methods) with the `--reconstruction_method` option.
+
+!!! tip
```
Comment (AliceJoubert) [inline docs/Pipelines/PET_Linear.md:46]:
```suggestion
!!! warning
```

File: docs/Pipelines/PET_Volume.md [hunk 3]
```diff
@@ -35,18 +35,24 @@ clinica run pet-volume [OPTIONS] BIDS_DIRECTORY CAPS_DIRECTORY GROUP_LABEL ACQ_L
...
 
 Pipeline options:
 
-- `--pvc_psf_tsv`: TSV file containing the `psf_x`, `psf_y` and `psf_z` of the PSF for each PET image.
-More explanation is given in [PET Introduction](../PET_Introduction) page.
+- `--pvc_psf_tsv`: TSV file containing the `psf_x`, `psf_y` and `psf_z` of the PSF for each PET image. More explanation is given in [PET Introduction](./PET_Introduction.md#partial-volume-correction-pvc) page.
 - `--smooth`: a list of integers specifying the different isotropic full width at half maximum ([FWHM](../glossary.md#fwhm)) in millimeters to smooth the image. Default value is: 0, 8 (both without smoothing and with an isotropic smoothing of 8 mm)
+- `--reconstruction_method`: Select only images based on a [specific reconstruction method](./PET_Introduction.md#reconstruction-methods).
+
+!!! tip
```
Comment (AliceJoubert) [inline docs/Pipelines/PET_Volume.md:51]:
```suggestion
!!! warning
```

**Review by AliceJoubert — APPROVED** [review]
LGTM, thanks @NicolasGensollen ! :-)

File: docs/Pipelines/PET_Introduction.md (hunk 1 above)
Comment (NicolasGensollen) [reply docs/Pipelines/PET_Introduction.md:52]: I have added a link to the ADNI documentation for PET images which explains this (see commit [d523bd8](https://github.com/aramis-lab/clinica/pull/1300/commits/d523bd80d15dcabff1b42d30a9e2b073034b7740)).

**Pull request merged**